# Matches a deprecation in more current versions of libbfio.  Otherwise, the
# stable libewf (20100226) doesn't link against it.  Borrowed from
# libewf-20100805
--- libewf-20100226.orig/liberror/liberror_definitions.h	2009-10-17 01:00:25.000000000 -0600
+++ libewf-20100226/liberror/liberror_definitions.h	2010-08-10 06:59:09.626742175 -0600
@@ -115,9 +115,11 @@
 
 	/* The de/uncompression failed
 	 */
-	LIBERROR_COMPRESSION_ERROR_UNCOMPRESS_FAILED   = 2
+	LIBERROR_COMPRESSION_ERROR_DECOMPRESS_FAILED   = 2
 };
 
+#define LIBERROR_COMPRESSION_ERROR_UNCOMPRESS_FAILED   LIBERROR_COMPRESSION_ERROR_DECOMPRESS_FAILED
+
 /* The encryption error codes
  * to signify errors regarding encryption
  */
